### PR TITLE
fix margin in back button

### PR DESCRIPTION
**Pull request by LiorSoffer**
resolve [FLPATH-1884](https://issues.redhat.com/browse/FLPATH-1884)

### CODE REVIEW

**Review by batzionb — APPROVED** [review]

File: workspaces/orchestrator/plugins/orchestrator-form-react/package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
   "dependencies": {
     "@backstage/core-components": "^0.15.1",
     "@backstage/core-plugin-api": "^1.10.0",
+    "@backstage/types": "^1.1.1",
```
Comment (batzionb) [inline workspaces/orchestrator/plugins/orchestrator-form-react/package.json:38]: I missed this in the review
Can you please revert this change?

Comment (LiorSoffer) [reply workspaces/orchestrator/plugins/orchestrator-form-react/package.json:38]: I can't pass the prettier tests without this change

Comment (batzionb) [reply workspaces/orchestrator/plugins/orchestrator-form-react/package.json:38]: I'll take a look to understand why
Meanwhile leave it as is
